### ISSUE TITLE
Docs: disable opening in playground for flow/typescript samples

### ISSUE THREAD

**Issue by srg-kostyrko** [JSMonk/hegel#163]
I opened accidentally in playground sample describing flow syntax and first thought that there is some error because in comments it was pointing that everything should pass but actually an error was reported

So maybe it would be a good idea to disable playground button for samples that describe flow or typescript

**Comment by JSMonk** [comment]
Yes, sounds great. Will disable the examples. Thank you for your contribution ^_^.

**Comment by thecotne** [comment]
or maybe link correct playground?
